src/nvim/os/unix_defs.h

```diff
@@ -31,12 +31,6 @@
 #ifndef USR_EXRC_FILE
 # define USR_EXRC_FILE "~/.exrc"
 #endif
-#ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE "~/.nvimrc"
```

Comment:
In https://github.com/neovim/neovim/issues/78 @ZyX-I made the case that there is no reason for us to support both old and new config locations. One thing that bothers me about it is that it means the default location is different on each system.

I would be less bothered by it if we defaulted to `~/.config` on all systems (unix, windows, osx) instead of the terrible `~/Library` and `%APPDATA` paths. No one _really_ wants their configs there, do they?